filenames.gypi

```diff
@@ -292,6 +293,7 @@
       'atom/common/api/atom_api_native_image.cc',
       'atom/common/api/atom_api_native_image.h',
       'atom/common/api/atom_api_native_image_mac.mm',
+      'atom/common/api/atom_api_process_stats.cc',
```

Comment:
Electron's extensions to `process` object are implemented in `atom/common/api/atom_bindings.cc`, so the code can be moved there.